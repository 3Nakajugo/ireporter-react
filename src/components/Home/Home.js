import React, { Component } from 'react';
import '../../assests/styles/home.css';

class Home extends Component {
  handleClick = (e) => {
    e.preventDefault();
    document.location.href = '/signup';
  };

  render() {
    return (
      <div>
        <div>
          <h1> iReporter</h1>
          <div className="intro">
            <p>
              iReporter is an online platform that enables any/every citizen to
              <br />
              bring any form of corruption to the notice of appropriate authorities and the
                    general public.
            </p>
            <div className="index-btn">
              <form>
                <div className="start-btn">
                  <input type="button" value="Get Started" onClick={this.handleClick} />
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    );
  }
}
export default Home;
