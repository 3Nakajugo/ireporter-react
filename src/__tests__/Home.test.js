import React from 'react';
import { shallow } from 'enzyme';
import Home from '../components/Home/Home';

describe('<Home />', () => {
  it('it should render the app', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
