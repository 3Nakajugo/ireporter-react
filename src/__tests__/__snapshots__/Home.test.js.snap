// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> it should render the app 1`] = `ShallowWrapper {}`;
